sysconfig/cloudinit.go

```diff
@@ -97,31 +97,40 @@ func configureCloudInit(opts *Options) (err error) {
 		return fmt.Errorf("unable to configure cloud-init, missing target dir")
 	}
 
-	// TODO: too naive to just re-use GadgetDir here automatically?
+	// first check if cloud-init should be disallowed entirely
+	if !opts.AllowCloudInit {
+		if err := DisableCloudInit(WritableDefaultsDir(opts.TargetRootDir)); err != nil {
```

Comment:
this could just be `return Disable ...` ? no?

Reply:
ah yes it should be sorry missed that